tests/labhub_test.py

```diff
@@ -155,21 +163,41 @@ def test_create_issue_cmd(self):
         labhub.REPOS = {'repository': self.mock_repo,
                         'repository.github.io': self.mock_repo}
 
-        testbot_public.assertCommand('!new issue repository this is the title\nbo\ndy',
-                              'Here you go')
+        testbot_public.assertCommand(
+            textwrap.dedent('''\
+                !new issue repository this is the title
+                first line of body
+                second line of body
+            '''),
+            'Here you go')
 
         labhub.REPOS['repository'].create_issue.assert_called_once_with(
-            'this is the title', 'bo\ndy\nOpened by @None at [text]()'
+            'this is the title',
+            textwrap.dedent('''\
+                first line of body
+                second line of body
+                Opened by @None at [text]()'''
+                            )
```

Comment:
Improper indentation.
Not same style as you use directly above

Reply:
@jayvdb  I had to do like this, otherwise it would have been `bo\ndy\nOpened by @None at [text]()\n`, with an unnecessary `\n` at the end, resulting in assertion error. Same for the one below.

Reply:
Irrelevant. Focus on the closing round bracket